### PR TITLE
PUD-317 - latest release info

### PR DESCRIPTION
**Pull request by jonnywyatt**
https://dsdmoj.atlassian.net/browse/PUD-317

- remove redundant 'addRecallType' and replace with 'last released' details page
- format dates going to / from the API
- safer date rendering
- back links on 'book recall' pages
- show either saved values or invalid values in forms

image

### CODE REVIEW

File: fake-manage-recalls-api/stubs/__files/get-recall.json [hunk 1]
```diff
@@ -10,5 +10,7 @@
     }
   ],
   "revocationOrderId": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
-  "recallEmailReceivedDateTime": "2020-12-05T15:33:57.000Z"
+  "recallEmailReceivedDateTime": "2020-12-05T15:33:57.000Z",
+  "lastReleasePrison": "Belmarsh",
+  "lastReleaseDateTime": "2020-08-03T00:00:00.000Z"
```
Comment (philsegal) [inline fake-manage-recalls-api/stubs/__files/get-recall.json:15]: nit pick alert: jira and title say 'latest' this calls it 'last'. I think we should be consistent

File: server/routes/handlers/book/lastRelease.test.ts [hunk 2]
```diff
@@ -1,34 +1,38 @@
 // @ts-nocheck
```
Comment (philsegal) [inline server/routes/handlers/book/lastRelease.test.ts:1]: Do we no longer need 'addRecallType'?

Comment (jonnywyatt) [reply server/routes/handlers/book/lastRelease.test.ts:1]: No, it's gone from the wireframes, it'll be calculated automatically based on sentence length

File: fake-manage-recalls-api/stubs/__files/get-recall.json (hunk 1 above)
Comment (jonnywyatt) [reply fake-manage-recalls-api/stubs/__files/get-recall.json:15]: the page title said 'last released' so I went with that

Comment (jonnywyatt) [reply fake-manage-recalls-api/stubs/__files/get-recall.json:15]: https://miro.com/app/board/o9J_l_ofeHo=/?moveToWidget=3074457361928159243&cot=14

**Review by philsegal — APPROVED** [review]

**Pull request merged**